tjreports/administrator/models/fields/filemultiple.php

```diff
@@ -8,9 +8,8 @@
  */
 
 defined('JPATH_BASE') or die;
-use Joomla\CMS\Form\FormField;
 
-jimport('joomla.form.formfield');
+use Joomla\CMS\Form\FormField;
```

Comment:
is it getting used?

i see
class JFormFieldFileMultiple extends JFormField

Reply:
namespace used to load the class in J4, this is needed to male code compatible with j3 and j4